### ISSUE TITLE
Feature-Request: Dark mode

### ISSUE THREAD

**Issue by homer1993** [sissbruecker/linkding#139]
I really enjoy the simplicity of linkding and like to use it in my docker environment. 
From a user point of view, I would like to use the page in a dark mode. 
Mabye that is something Im not alone with :)

**Comment by thelittlerocket** [comment]
Hi @homer1993 
Do you mean this?
<img width="285" alt="linkding 2021-08-13 at 8 05 42 AM" src="https://user-images.githubusercontent.com/75967438/129285080-04b698ab-9673-48b9-ad4a-36104bf2d770.png">

**Comment by homer1993** [comment]
Hi @thelittlerocket 
Thanks for the fast reaction. I have recognized, that I use the wrong dockerhub image. 
So far, I used tianbin/linkding which is obviously the wrong image. So, Ive created a new stack following the [docker compose file](https://github.com/sissbruecker/linkding/blob/master/docker-compose.yml) from the project and saw several nice changes including dark mode. Awesome, I love it. 
Thanks alot for doing this project.
